lib/rules/media-feature-name-value-whitelist/README.md

```diff
@@ -0,0 +1,65 @@
+# media-feature-name-value-whitelist
+
+Specify a whitelist of allowed media feature and value pairs.
+
+```css
+@media screen and (min-width: 768px) {}
+/**                    ↑        ↑
+ *         These features and these values */
+```
+
+**Caveat:** Media feature names within range or boolean context are ignored.
+
+## Options
+
+```json
+{
+  "max-width": ["Array", "of", "values"],
+  "/regex/": ["/regex/", "non-regex"]
+}
+```
+
+If a media feature is found, only its whitelisted property values are allowed. The rule reports all
+non-matching values.
+
+If a property or value is surrounded with `/` (e.g. `"/width$/"`), it is interpreted as a regular
+expression. For example, `/width$/` will match `max-width` and `min-width`.
+
+## Examples
+
+Given
+
+```json
+{
+  "min-width": ["768px"],
```

Comment:
I think two values here would be useful to the reader e.g. `"min-width": ["768px", "1024px"]`.

You'll need to add an additional pattern to the "*not* considered violations" list for it.